VideoLocker/src/main/java/org/edx/mobile/base/BaseSingleFragmentActivity.java

```diff
@@ -131,24 +131,16 @@ public void finishProcess(){
         hideLoadingProgress();
     }
 
-    public void onMessage(@NonNull MessageType messageType, @NonNull String message){
+    public void onMessage(@NonNull MessageType messageType, @NonNull String message) {
         //TODO - -we need to define different UI message view for different message type?
-        switch ( messageType ){
+        switch (messageType) {
             case FLYIN_ERROR:
```

Comment:
I guess, we can leave the deprecation for now, as we will be overhauling these messages, their types and how the errors are shown in the near future.

Having different types might come in handy.

thoughts @1zaman, cuz you are working on the Version based app upgrade which involves some modifications to the these \*flying things\*.